tools/cve/html/body-part.tpl

```diff
@@ -0,0 +1,52 @@
+{{- if . }}
+      <table>
+      {{- range . }}
+        <tr class="group-header"><th colspan="6">{{ escapeXML .Type }}</th></tr>
+        {{- if (eq (len .Vulnerabilities) 0) }}
+        <tr><th colspan="6">No Vulnerabilities found</th></tr>
+        {{- else }}
+        <tr class="sub-header">
+          <th>Package</th>
+          <th>Vulnerability ID</th>
+          <th>Severity</th>
+        </tr>
+          {{- range .Vulnerabilities }}
+        <tr class="severity-{{ escapeXML .Vulnerability.Severity }}">
+          <td class="pkg-name">{{ escapeXML .PkgName }}</td>
+          <td>
+            <a href={{ escapeXML (index .Vulnerability.References 0) | printf "%q" }}>{{ escapeXML .VulnerabilityID }}</a>
+          </td>
+          <td class="severity">{{ escapeXML .Vulnerability.Severity }}</td>
+        </tr>
+          {{- end }}
+        {{- end }}
+        {{- if (eq (len .Misconfigurations ) 0) }}
+        <tr><th colspan="6">No Misconfigurations found</th></tr>
+        {{- else }}
+        <tr class="sub-header">
+          <th>Type</th>
+          <th>Misconf ID</th>
+          <th>Check</th>
+          <th>Severity</th>
+          <th>Message</th>
+        </tr>
```

Comment:
'Vulnerabilities' have 3 columns and 'Misconfigurations' have 5 columns, it seems another table is required.